.gitignore

```diff
@@ -33,6 +33,7 @@ linux/netfilter/vendor/
 __pycache__
 lottie.mjs
 qml_lint_result.txt
+qtglean/Cargo.lock
```

Comment:
because it has been committed, you have to remove it too.

Reply:
:doh: